community_version.py

```diff
@@ -1,6 +1,7 @@
 # Community Version
 import argparse
 import webbrowser
+image re
```

Comment:
I'm guessing this should be `import re`.

Reply:
ohh yes, doing multiple time ..i have missed it

Reply:
like below image
<img width="1107" alt="Screen Shot 2022-10-11 at 10 04 35 AM" src="https://user-images.githubusercontent.com/109090770/195446525-a9e94acd-ed50-4f82-9f87-36d41fec0b7d.png">